test/functional/wallet_basic.py

```diff
@@ -158,6 +158,9 @@ def run_test(self):
         spent_0 = {"txid": node0utxos[0]["txid"], "vout": node0utxos[0]["vout"]}
         assert_raises_rpc_error(-8, "Invalid parameter, expected unspent output", self.nodes[0].lockunspent, False, [spent_0])
 
+        # Verify that a spent output can be unlocked
+        assert_equal(self.nodes[0].lockunspent, True, [spent_0], True)
```

Comment:
Seems like this is an improper use of `assert_equal` - basically it's meant to be use with arg1 == all other args, so the two Trues are redundant or mistaken.

https://github.com/bitcoin/bitcoin/blob/master/test/functional/test_framework/util.py#L36